### DISCUSSION TITLE
Pycaret in powerbi

### DISCUSSION THREAD

**Discussion by azizallouch** [pycaret/pycaret#3705] [Q&A]
can pycaret be deployed in power bi service?

**Comment by PabloJMoreno** [comment]
Of course! 
If you are using it to create visuals, you will need a Pro license. 
If you are using it at Power Query, you will need a gateway in 'personal mode' and jsut basic free license will sufice.

Thank you

Take a look: 
https://www.linkedin.com/pulse/integrating-pycaret-time-series-module-power-bi-pablo-moreno/?trackingId=IPNekhwkSdumbbxNXpp%2FYg%3D%3D 

https://www.linkedin.com/pulse/creating-machine-learning-app-power-bi-pycaret-panama-user-group/?trackingId=IPNekhwkSdumbbxNXpp%2FYg%3D%3D
